docs/conf.py

```diff
@@ -16,5 +16,5 @@
 project = module.__name__
 copyright = ', '.join((module.metadata['copyright'], module.metadata['author']))
 author = module.metadata['author']
-version = module.__version__
-release = module.__version__
+version = str(module.__version__)
```

Comment:
This looks like it should have gone into the incremental PR.  Can you back this out and do a new PR for it?